locksmith/.eslintignore

```diff
@@ -0,0 +1,8 @@
+/build
+/coverage
+/__tests__/utils/withConfig/__snapshots
+/src/.next
+/src/out
+/.cache
+/models
```

Comment:
Hum. You probably want `/models` to be eslint-ed? Also please cleanup the `/build`, ` /__tests__/utils/withConfig/__snapshots`, ` /src/*` since they're not applicable here?

Reply:
Thanks for pointing this out, I added `models` and `migrations` here because the content was autogenerated. Will clean up the other bits